applications/luci-app-cloudflared/htdocs/luci-static/resources/view/cloudflared/base.js

```diff
@@ -0,0 +1,85 @@
+/* SPDX-License-Identifier: GPL-3.0-only
+ *
+ * Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+ */
+
+'use strict';
+'require form';
+'require poll';
+'require rpc';
+'require uci';
+'require view';
+
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getServiceStatus() {
+	return L.resolveDefault(callServiceList('cloudflared'), {}).then(function (res) {
+		var isRunning = false;
+		try {
+			isRunning = res['cloudflared']['instances']['cloudflared']['running'];
+		} catch (e) { }
+		return isRunning;
+	});
+}
+
+function renderStatus(isRunning) {
+	var spanTemp = '<label class="cbi-value-title">Status</label><div class="cbi-value-field"><em><span style="color:%s">%s</span></em></div>';
+	var renderHTML;
+	if (isRunning) {
+		renderHTML = String.format(spanTemp, 'green', _('Running'));
+	} else {
+		renderHTML = String.format(spanTemp, 'red', _('Not Running'));
+	}
+
+	return renderHTML;
+}
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			uci.load('cloudflared')
+		]);
+	},
+
+	render: function(data) {
+		var m, s, o;
+
+		m = new form.Map('cloudflared', _('Cloudflared'),
+			_('Zero Trust Security services from Cloudflare to help you get maximum security both from outside and within the network. <br/><br/><a href="https://github.com/animegasan" target="_blank">Powered by animegasan</a>'));
```

Comment:
Do we really need the `Powered by animegasan` here? All translators will have to translate this in all languages,

Reply:
Please remove the "powered by animegasan" advertisement. LuCI apps do not advertise the creator.
 And here "powered by you"  looks even more strange, as you are not the author of the actual cloudflared package or the underlying application, right?

Reply:
It's already removed